internal/pkg/cli/pipeline_init.go

```diff
@@ -20,36 +25,39 @@ import (
 	"github.com/aws/copilot-cli/internal/pkg/deploy/cloudformation"
 	"github.com/aws/copilot-cli/internal/pkg/manifest"
 	"github.com/aws/copilot-cli/internal/pkg/template"
-	"github.com/aws/copilot-cli/internal/pkg/term/color"
 	"github.com/aws/copilot-cli/internal/pkg/term/command"
-	"github.com/aws/copilot-cli/internal/pkg/term/log"
 	"github.com/aws/copilot-cli/internal/pkg/term/prompt"
-	"github.com/aws/copilot-cli/internal/pkg/version"
 	"github.com/aws/copilot-cli/internal/pkg/workspace"
 	"github.com/spf13/afero"
-	"github.com/spf13/cobra"
 )
 
 const (
 	pipelineSelectEnvPrompt     = "Which environment would you like to add to your pipeline?"
 	pipelineSelectEnvHelpPrompt = "Adds an environment that corresponds to a deployment stage in your pipeline. Environments are added sequentially."
 
-	pipelineSelectGitHubURLPrompt     = "Which GitHub repository would you like to use for your service?"
-	pipelineSelectGitHubURLHelpPrompt = `The GitHub repository linked to your workspace.
+	pipelineSelectURLPrompt     = "Which repository would you like to use for your service?"
+	pipelineSelectURLHelpPrompt = `The repository linked to your workspace.
```

Comment:
Im not entirely sure but should these be pipeline?
```suggestion
	pipelineSelectURLPrompt     = "Which repository would you like to use for your pipeline?"
	pipelineSelectURLHelpPrompt = `The repository linked to your pipeline.
```

Reply:
Good catch! Fixed.